tests/helper.py

```diff
@@ -1,6 +1,8 @@
-import os, subprocess, shutil, tempfile
+import os
+import subprocess
+import shutil
+import tempfile
 from git import Repo, GitCommandError
-from testpath import MockCommand, modified_env
```

Comment:
These were not used, sorry for whitespace change for the file that `ruff` caused.